app/controllers/signup.js

```diff
@@ -2,33 +2,20 @@ import Controller from '@ember/controller';
 import { action, set } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
-import registerUser from '../utils/register-api';
 import { GOTO_URL } from '../constants/signup';
-import { NEW_SIGNUP_FLOW, OLD_SIGNUP_FLOW } from '../constants/analytics';
-import ENV from 'website-my/config/environment'; // remove this when new flow goes live
+import { OLD_SIGNUP_FLOW } from '../constants/analytics';
+import ENV from 'website-my/config/environment';
 
-const BASE_URL = ENV.BASE_API_URL; // remove this when new flow goes live
+const BASE_URL = ENV.BASE_API_URL;
 
 export default class SignupController extends Controller {
   @service analytics;
 
-  queryParams = ['state', 'dev'];
-
   @tracked isSubmitClicked = false;
-  @tracked isButtonDisabled = true;
-  @tracked isSubmitDisabled = true; // remove this when new flow goes live
+  @tracked isSubmitDisabled = true;
```

Comment:
Why is this not derived? Where do we use it later? What changes it, as I am not seeing an `@action` to go with it.